packages/react-magma-dom/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -69,23 +111,26 @@ export const NativeSelect = React.forwardRef<HTMLDivElement, NativeSelectProps>(
         messageStyle={messageStyle}
         ref={ref}
       >
-        <SelectTriggerButton
+        <StyledNativeSelectWrapper
           disabled={disabled}
           hasError={!!errorMessage}
           isInverse={isInverse}
-          toggleButtonProps={''}
+          theme={theme}
         >
           <StyledNativeSelect
+            aria-label={nativeSelectLabel}
             {...other}
```

Comment:
This works, but it may be nice to be a little more explicit about the aria label.

Under props, we should declare the aria-label (`'aria-label': ariaLabel,`) and then we can have the aria-label conditionally have either the one passed by the user if it exists, or fall back on the default:
```
{...other}
aria-label={ariaLabel || nativeSelectLabel}
```

Reply:
Removed from component as deemed unnecessary by Leslie.